### PR TITLE
Update to allow for member profile URLs to work using member slug

### PR DESCRIPTION
**Pull request by martyndavies**
Changes made:
- Switch over to using the member slug in profile URLs to handle merge cases where the GitHub username may not persist as the main slug
- Switch add comment functionality over to use slug as part of member URL
- Update `getMemberContributions` fetch URL to include the `source=github` and `username` attributes as per latest API spec
- Added fix for 500 error when manually adding the comment to a user's timeline

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "Orbit",
-  "version": "0.3.2",
+  "version": "0.3.3",
```
Comment (phacks) [inline manifest.json:4]: There’s a few more instances where the version number needs bumping—you can look at [this commit](https://github.com/orbit-love/orbit-browser-extension/commit/584a574893294006a3db5dfd0b48b793504c39df) for reference

Comment (phacks) [reply manifest.json:4]: I usually apply the bump after all other commits, then do a git tag and a github release. We can look at this together if you’d like (in addition to pushing the update to the Chrome Web Store)

File: src/github/orbit-helpers.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ export const orbitAPI = {
   async getMemberContributions(ORBIT_CREDENTIALS, username) {
     try {
       const response = await fetch(
-        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find?github=${username}&api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
+        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find?source=github&username=${username}&api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
```
Comment (phacks) [inline src/github/orbit-helpers.js:67]: Unrelated to the issue at hand, but we could work on another update to move away from URL-based authentication and use the Authorization header instead

File: src/github/orbit-helpers.js [hunk 3]
```diff
@@ -243,12 +243,15 @@ export const orbitAPI = {
   ) {
     try {
       const response = await fetch(
-        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/${member}/posts?api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
+        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/${member}/activities?api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
         {
           method: "POST",
           body: JSON.stringify({
```
Comment (phacks) [inline src/github/orbit-helpers.js:249]: I see you’re creating a custom activity here, but I think what we want (and used to have) is a Content Activity.

The idea there was to allow “bookmarking” as Content comments on repos outside of your organization. For repos in your organization, the “IssueCommentActivity” will be created by the GH integration automatically. I fear that a `issue_comment:created` custom activity might create confusion with the “native” issue comment activity

File: src/github/orbit-helpers.js (hunk 2 above)
Comment (martyndavies) [reply src/github/orbit-helpers.js:67]: I took a look at that as an option yesterday and ran into CORS issues when switching to Bearer-based authentication. Will investigate further.

File: src/github/orbit-helpers.js (hunk 3 above)
Comment (martyndavies) [reply src/github/orbit-helpers.js:249]: Yes, you're right. I took a guess here.

File: src/github/orbit-helpers.js (hunk 2 above)
Comment (phacks) [reply src/github/orbit-helpers.js:67]: Hah—CORS issues, those are the worst. The following screenshot was taken when I was working on this extension:

10 commit messages ranting about damn CORS

Happy to take a look with you on this!

File: src/github/orbit-action.test.js [hunk 4]
```diff
@@ -208,12 +208,14 @@ test("should create content for existing members", async () => {
...
-    expect.stringContaining("/my-workspace/members/phacks/posts"),
+    expect.stringContaining("/my-workspace/members/phacks/activities"),
     expect.objectContaining({
       body: JSON.stringify({
-        url:
-          "https://github.com/orbit-love/orbit-model/issues/10#issuecomment-590037251",
-        published_at: "2020-02-23T07:55:28Z",
+        activity_type: "content",
+        url: "https://github.com/orbit-love/orbit-model/issues/10#issuecomment-590037251",
+        title: "Commented on GitHub",
```
Comment (phacks) [inline src/github/orbit-action.test.js:216]: I believe `title` and `description` are not supported for ContentActivities—the “fancy” display in the UI is generated by a link thumbnailer in the backend

**Review by phacks — COMMENTED** [review]
Could you look whether this `gitHubUsername` should be replaced by `$slug` as well?

https://github.com/orbit-love/orbit-browser-extension/blob/bd4e23e92111788fef75545bfc3eb6afff7c346e/src/github/orbit-action.js#L340-L351

Otherwise all looks good!

**Review by phacks — APPROVED** [review]
I think this is ready to merge! We can look into the version bump + deployment process together if you’d like

**Pull request merged**